src/reactviews/pages/SchemaCompare/components/SchemaDifferences.tsx

```diff
@@ -101,67 +151,163 @@ const SchemaDifferences = ({ onDiffSelected }: Props) => {
+                <TableCell>{formatName(item.sourceValue)}</TableCell>
+                <TableCell>
+                    <Checkbox
+                        checked={item.included}
+                        onClick={() => handleIncludeExcludeNode(item, !item.included)}
+                        onKeyDown={(e) => toggleKeyDown(e, item, !item.included)}
+                    />
+                </TableCell>
+                <TableCell>{getLabelForAction(item.updateAction as number)}</TableCell>
+                <TableCell>{formatName(item.targetValue)}</TableCell>
             </TableRow>
         );
     };
 
     return (
-        <>
-            {compareResult?.success && (
-                <Table
-                    noNativeElements
-                    aria-label="Table with schema differences"
-                    aria-rowCount={compareResult.differences.length}
-                    style={{ minWidth: "550px" }}>
-                    <TableHeader>
-                        <TableRow aria-rowindex={1}>
-                            {columns.map((column) => (
-                                <TableHeaderCell>{column.renderHeaderCell()}</TableHeaderCell>
-                            ))}
-                        </TableRow>
-                    </TableHeader>
-                    <TableBody>
-                        <List
-                            height={200}
-                            itemCount={items.length}
-                            itemSize={45}
-                            width={"100%"}
-                            itemData={items}>
-                            {RenderRow}
-                        </List>
-                    </TableBody>
-                </Table>
-            )}
-        </>
+        <Table
+            noNativeElements
+            aria-label="Table with selection"
+            aria-rowcount={rows.length}
+            style={{ minWidth: "650px" }}>
+            <TableHeader>
+                <TableRow aria-rowindex={1}>
+                    <TableHeaderCell className={classes.HeaderCellPadding}>
+                        {loc.schemaCompare.type}
+                    </TableHeaderCell>
+                    <TableHeaderCell className={classes.HeaderCellPadding}>
+                        {loc.schemaCompare.sourceName}
+                    </TableHeaderCell>
+                    <TableHeaderCell className={classes.PrimaryCheckboxPadding}>
+                        {!context.state.isIncludeExcludeAllOperationInProgress && (
+                            <Checkbox
+                                checked={
+                                    allDiffsIncluded ? true : someDiffsExcluded ? "mixed" : false
+                                }
+                                onClick={() => handleIncludeExcludeAllNodes()}
+                                onKeyDown={toggleAllKeydown}
+                            />
+                        )}
+                        {context.state.isIncludeExcludeAllOperationInProgress && (
```

Comment:
if this operation is in progress, the other checkboxes should be disabled; otherwise this could easily get messed up.